Dockerfile

```diff
@@ -48,7 +48,9 @@ RUN pecl install igbinary-1.2.1 && \
 
 # Prevent newrelic daemon from auto-spawning; uses newrelic run.d script to enable at runtime, when ENV variables are present
 # @see https://docs.newrelic.com/docs/agents/php-agent/advanced-installation/starting-php-daemon-advanced
-RUN sed -i "s/;newrelic.daemon.dont_launch = 0/newrelic.daemon.dont_launch = 3/" /etc/php5/mods-available/newrelic.ini
+RUN sed -i "s/;newrelic.daemon.dont_launch = 0/newrelic.daemon.dont_launch = 3/" /etc/php5/mods-available/newrelic.ini && \
+    sed -i "s/listen = \(.*\)\+/listen = 127.0.0.1:9000/" /etc/php5/fpm/pool.d/www.conf
```

Comment:
right, other than the health check, there is no load.

I was thinking if the cause might be related with the selfie checks and the requests not being terminated properly, which would leave the process running for more time, resulting on all children busy and no more of them available for nginx.

When we docker-entered the container, the load was high (above 1.00) for the last 5 and 15 minutes, which doesn't make any sense with that little traffic.

Just a thought.